### PR TITLE
removed tabster dependency.

### PR DESCRIPTION
**Pull request by Anush2303**
## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.template.ts [hunk 1]
```diff
@@ -1,13 +1,7 @@
 import { ElementViewTemplate, html, ref, repeat, when } from '@microsoft/fast-element';
-import { createTabster, getGroupper, getMover } from 'tabster';
```
Comment (AtishayMsft) [inline packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.template.ts:2]: what about donut chart and tests?

Comment (Anush2303) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.template.ts:2]: In donut chart also, tabster will not work

Comment (Anush2303) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.template.ts:2]: Ohh okay, will remove from there as well

Comment (Anush2303) [reply packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.template.ts:2]: Not used in tests

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**